Makefile

```diff
@@ -41,6 +41,13 @@ count_lines:
         '{printf "%4s %s\n", $$1, $$2}{s+=$$0}END{print s}'
 	@echo ''
 
+
+# ----------------------------------
+#       STREAMLIT RUN LOCAL
+# ----------------------------------
+run_streamlit:
+    streamlit run app.py
```

Comment:
is it app.py or chessapi.py here? we don't have an app.py so far

Reply:
app.py will eventually be the streamlit front end file, not the api. Sorry I added it to the Makefile ahead of time.
you can sping the api locally with
uvicorn chessapi:app --reload